icelandblog.html

```diff
@@ -0,0 +1,209 @@
+<!doctype html>
+<html lang="en">
+
+
+<head>
+
+  <!--  TAGS THAT DESCRIBE YOUR PAGE AND LET'S THE BROWSER KNOW SOME BASIC INFO-->
+  <meta charset="utf-8">
+  <meta name="description" content="1-sarah shaw">
+  <title>Basic Structure</title>
+
+  <!-- SOME MORE META TAGS-->
+  <meta name="keywords" content="HTML, CSS, blogpost">
+  <meta name="author" content="sarah shaw: sarah went solo">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+ <link rel="stylesheet" type="text/css" href="mycss.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+</head>
+
+<body>
+
+    <!-- HEADER WITH MY NAV and logo image-->
+
+        <div class="logo">
+            <nav>
+
+                <img src="images/logo.png" style="width:30px height:auto;" alt="">
+
+            </nav>
+
+
+        </div>
+        <div class="navigation monfont">
+            <nav>
+
+                    <a href="#">About Me</a>
```

Comment:
:hammer: @sarahfshaw  looks like you forgot to link out your index.html page here